data/data_downloader.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+import pandas as pd
+from astroquery.mast import MastMissions, MastMissionsClass
+
+SECTORS = range(1, 27)
+
+
+def get_toi_df() -> pd.DataFrame:
```

Comment:
```suggestion
def get_toi_df(url: str, cache_enabled: bool = True) -> pd.DataFrame:
```

Change the function body accordingly.

Reply:
It's just one time download. There's no need for cache_enabled parameter